Wit.ai.net.Tests/WitConversation_Tests.cs

```diff
@@ -33,7 +35,7 @@ public void doSay(string conversationId, object context, string msg, double conf
             Console.WriteLine(msg);
         }
 
-        public object doAction(string conversationId, object context, string action, double confidence)
```

Comment:
I'm not understood how to use doAction() and update context every time.please let me know asp.
Thanks